_posts/2025-04-11-Hybrid-Search-Options-And-Practice.md

```diff
@@ -1,30 +1,35 @@
 ---
 layout: post
-title:  Hybrid Search with OpenSearch: Options and Practice
+title:  "Building effective hybrid Search in OpenSearch: Techniques and best practices"
 authors:
   - kazabdu
-  - xiaoyu
-date: 2025-04-08
+  - seanzheng
+  - kolchfa
+date: 2025-04-11
 has_science_table: true
 categories:
   - technical-posts
 meta_keywords: z-score normalization, OpenSearch, neural search plugin, hybrid search, relevance ranking, search normalization, k-nn search, min-max normalization, how reciprocal rank fusion works
-meta_description: Learn about different searches in OpenSearch. Discover how hybrid search merges results from multiple query sources for improved relevance.
+meta_description: Learn about lexical, semantic, and hybrid search in OpenSearch. Discover how hybrid search merges results from multiple query sources for improved relevance.
 ---
 
-In the evolving world of search technology, OpenSearch stands out with its powerful search capabilities. OpenSearch offers a range of search techniques, starting with basic lexical search using BM25 (Best Matching 25), which excels at keyword-based retrieval and remains a cornerstone of many search applications. The evolution continues with semantic search, which leverages dense vector embeddings to capture the semantic meaning beyond keywords. Advanced hybrid techniques combine lexical and semantic approaches, using normalization methods like score-based and rank-based algorithms to balance and integrate scores from different search methods. These techniques progressively enhance search relevance and flexibility, catering to diverse search requirements in modern applications. This blog post will guide you through this series of search techniques, from BM25 to hybrid search and provide an overview of how Hybrid Search works in OpenSearch.
+In the evolving world of search technology, OpenSearch provides a wide range of capabilities to support modern search needs. It starts with traditional keyword-based techniques, like lexical search using BM25 (Best Matching 25), and expands to more advanced methods like semantic and hybrid search. BM25 remains a reliable and efficient choice for many applications that rely on keyword relevance. Semantic search builds on this by using machine learning to capture meaning beyond keywords. Advanced hybrid techniques combine lexical and semantic approaches, using normalization methods like score-based and rank-based algorithms to balance and integrate scores from different search methods. These techniques progressively enhance search relevance and flexibility, catering to diverse search requirements in modern applications.
 
+In this blog post, we'll explore each of these techniques---lexical, semantic, and hybrid---and deep dive into how hybrid search works in OpenSearch.
 
-### 1. Lexical Search vs Semantic Search
+## Comparing lexical and semantic search
```

Comment:
Don't we want to keep it as Comparing Lexical and Semantic Search

Reply:
If you're referring to capitalization, then our style guide prescribes using sentence case (first word capitalized and subsequent words lowercase).